src/pages/workspaces/workspace/Data.js

```diff
@@ -716,7 +733,8 @@ const WorkspaceData = _.flow(
                   after: isDataTabRedesignEnabled() && h(DataTableActions, {
                     tableName: type,
                     rowCount: typeDetails.count,
-                    workspace
+                    workspace,
+                    onSuccess: () => loadMetadata()
```

Comment:
nit: Could we use a more specific name than "onSuccess"? Maybe: onUpdateSuccess()? Since this only called by rename metadata success, and not by the other menu items.

Reply:
Sure, addressed in 4cb07b8d2201e7b85f9d751de33e4a62fdc646e0